lib/bootloader_pvm.pm

```diff
@@ -201,6 +237,32 @@ sub boot_hmc_pvm {
 
     # don't wait for it, otherwise we miss the menu
     enter_cmd "mkvterm -m $hmc_machine_name --id $lpar_id";
+    assert_screen "hmc-openfirmware-prompt";
+    if (get_var('BOOT_CD_CMD')) {
```

Comment:
in this case, what would `BOOT_CD_CMD` resolve to? Also needs to be added to variables.md

Reply:
you can find it here ,https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19066/files#diff-eb176a275e164bf44779feb951ab9bbcbb60959e2b5227a4e23101962d69299e